docs/migrate/migrate-estimator2.rst

```diff
@@ -0,0 +1,211 @@
+Calculate expectation values in an algorithm
+==============================================
+
+The role of the `Estimator` primitive is two-fold: on one hand, it acts as an entry point to the quantum devices or simulators, replacing `backend.run()` (or `QuantumInstance.execute()`). On the other hand, it is an **algorithmic abstraction** for expectation value calculations, which removes the need to perform operations to construct the final expectation circuit. This results in a considerable reduction of the code complexity, and a more compact algorithm design. 
+
+The following example uses common tools from the ``qiskit.opflow`` module as a reference for the "legacy way of doing things", but we acknowledge that some of you might have used custom code for this task. In that case, you can decide between keeping your custom code and replacing `backend.run()` with a `Sampler`, or replacing your custom code with the `Estimator` primitive.
+
+
+Problem definition 
+-------------------------------
+
+We want to compute the expectation value of a quantum state (circuit) with respect to a certain operator. Here we are using the H2 molecule and an arbitrary circuit as the quantum state:
+
+
+.. code-block:: python
+
+
+    from qiskit import QuantumCircuit
+    from qiskit.quantum_info import SparsePauliOp
+
+    # Step 1: Define operator
+    op = SparsePauliOp.from_list(
+        [
+            ("II", -1.052373245772859),
+            ("IZ", 0.39793742484318045),
+            ("ZI", -0.39793742484318045),
+            ("ZZ", -0.01128010425623538),
+            ("XX", 0.18093119978423156),
+        ]
+    )
+
+    # Step 2: Define quantum state
+    state = QuantumCircuit(2)
+    state.x(0)
+    state.x(1)
+
+_2legacy-opflow:
+
+Legacy methods (using opflow)
+-----------------------------
+
+`Opflow <https://qiskit.org/documentation/apidoc/opflow.html>`__ provided its own classes to represent both operators and quantum states:
+
+.. code-block:: python
+
+    from qiskit.opflow import CircuitStateFn, PauliSumOp
+
+    opflow_op = PauliSumOp(op)
+    opflow_state = CircuitStateFn(state) # convert to a state
+
+New methods (using primitives)
+-------------------------------
+
+These code examples have been updated to use primitives.
+
+Opflow provided its own classes to represent both operators and quantum states:
+
+.. code-block:: python
+
+    from qiskit.opflow import CircuitStateFn, PauliSumOp
+
+    opflow_op = PauliSumOp(op)
+    opflow_state = CircuitStateFn(state) # convert to a state
+
```

Comment:
Here too, this part can be removed. I noticed that you added twisties, I think that it looks very nice :)
```suggestion
```